_data/projects/metasmoke.yml

```diff
@@ -0,0 +1,13 @@
+name: metasmoke
+desc: Web dashboard and back end for SmokeDetector, a spam-detecting bot
+site: *https://metasmoke.erwaysoftware.com/
+tags:
```

Comment:
@ArtOfCode- I am not able to get why Travis is failing. I guess it is due to these extra indentations in your tags. Please correct this. This should solve the travis failure.